cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -0,0 +1,277 @@
+define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
+        'common/js/spec_helpers/template_helpers', 'js/views/move_xblock_list',
+        'js/views/move_xblock_breadcrumb', 'js/models/xblock_info'],
+    function($, _, AjaxHelpers, TemplateHelpers, MoveXBlockListView, MoveXBlockBreadcrumbView,
+             XBlockInfoModel) {
+        'use strict';
+        describe('MoveXBlock', function() {
+            var renderViews, createXBlockInfo, createCourseOutline, moveXBlockBreadcrumbView,
+                moveXBlockListView, parentToChildMap, categoryMap, createChildXBlockInfo,
+                verifyBreadcrumbViewInfo, verifyListViewInfo, getDisplayedInfo, clickForwardButton,
+                clickBreadcrumbButton, verifyXBlockInfo, nextCategory;
+
+            parentToChildMap = {
+                course: 'section',
+                section: 'subsection',
+                subsection: 'unit',
+                unit: 'component'
+            };
+
+            categoryMap = {
+                section: 'chapter',
+                subsection: 'sequential',
+                unit: 'vertical',
+                component: 'component'
+            };
+
+            beforeEach(function() {
+                setFixtures(
+                    "<div class='breadcrumb-container'></div><div class='xblock-list-container'></div>"
+                );
+                TemplateHelpers.installTemplates([
+                    'move-xblock-list',
+                    'move-xblock-breadcrumb'
+                ]);
+            });
+
+            afterEach(function() {
+                moveXBlockBreadcrumbView.remove();
+                moveXBlockListView.remove();
+            });
+
+            createChildXBlockInfo = function(category, options, xblockIndex) {
+                var cInfo =
+                    {
+                        category: categoryMap[category],
+                        display_name: category + '_display_name_' + xblockIndex,
+                        id: category + '_ID'
+                    };
+
+                return createXBlockInfo(parentToChildMap[category], options, cInfo);
+            };
+
+            createXBlockInfo = function(category, options, outline) {
+                var cInfo =
+                    {
+                        category: categoryMap[category],
+                        display_name: category,
+                        children: []
+                    },
+                    xblocks;
+
+                xblocks = options[category];
+                if (!xblocks) {
```

Comment:
why don't we just check `!options[category]` instead of storing gin the variable ?